corehq/apps/case_importer/static/case_importer/js/import_history.js

```diff
@@ -19,7 +19,25 @@ hqDefine('case_importer/js/import_history.js', function () {
             // this prevents some jumpiness when not necessary
             // and is particularly bad if you're in the middle of editing a comment
             return !(_.chain(self.case_uploads()).pluck('upload_id').isEqual(_(data).pluck('upload_id')).value() &&
-                _.chain(self.case_uploads()).pluck('task_status').isEqual(_(data).pluck('task_status')).value());
+                _.chain(self.case_uploads()).pluck('task_status').map(function (task_status) {return task_status()})
+                    .isEqual(_(data).pluck('task_status').map(function (task_status) {return task_status()})).value());
+        };
+        self.updateCaseUploads = function (data) {
```

Comment:
Oh sorry I think I was a little unclear. I think `updateCaseUploads` is a fine name for the function, but I think the name `data` for the argument could be replaced with something that describes what sort of data is being passed to the function.